recruitment/models.py

```diff
@@ -101,10 +101,8 @@ def handle_answers_from_request(self, request, user):
                         user=user
                     )
                     answer_string = request.POST[key]
-
-                    if answer_string:
-                        answer.answer = answer_string
-                        answer.save()
+                    answer.answer = answer_string
+                    answer.save()
```

Comment:
I think it would be better to rewrite this as:
```Python
if key in request.POST:
    answer.answer = request.POST[key]
    answer.save()
```
Right now there's a risk of data being deleted by mistake if a partial form is ever submitted. That is, it seems dangerous to assume that a key being absent in `request.POST` means "delete all the data stored for that key" :)